packages/core/src/components/Label/Label.module.scss

```diff
@@ -147,6 +147,15 @@
       background-color: var(--positive-color-selected);
     }
   }
+
+  .smallText {
+    line-height: 14px;
+  }
+}
+
+.small {
+  padding: 0 var(--spacing-xs);
+  border-radius: 2px;
 }
```

Comment:
Let's nest it inside label, it will increase specificity

Reply:
Done